### PR TITLE
Add SearchBar component

### LINKED ISSUE #7: Create a Google search bar component
**User Story**
As a user, I want to be able to perform a Google search from the home page of the application.

**Acceptance Criteria**
* A search bar component - an input text box where a search query can be typed
* A button - when pressed, the google search query is opened in a new tab

**Additional Context**
Figma Link: https://www.figma.com/file/J2DDVa9redQigCzTee56xw/SE701-Daily-Focus?node-id=0%3A1

### PR DESCRIPTION
**Pull request by Denijar**
**Issue:**
The user story which describes a need to perform a google search which opens in a new tab has not been fulfilled.

**Solution:**
Add a SearchBar component which allows the user to perform a Google search.

**Risk:**
This component has not been tested. Please refer to Discord discussion about whether test cases must be written in this PR or if another issue will be made for testing.

**Reviewed By:**
[Edit the message as to who reviewed it here]

**Linked Issues**
Closes #7

### CODE REVIEW

File: client/src/components/SearchBar/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
```
Comment (sannan06) [inline client/src/components/SearchBar/index.js:1]: File name should be SearchBar.js

File: client/src/components/SearchBar/style.module.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.container {
```
Comment (sannan06) [inline client/src/components/SearchBar/style.module.css:1]: Can we use .scss files for styling going forward

**Review by sannan06 — CHANGES_REQUESTED** [review]

File: client/src/components/SearchBar/index.js (hunk 1 above)
Comment (Denijar) [reply client/src/components/SearchBar/index.js:1]: Nah naming it index.js means that you can import it from App.js with 'import SearchBar from './components/SearchBar'. If it was SearchBar.js then you would need './components/SearchBar/SearchBar.js'

File: client/src/components/SearchBar/style.module.css (hunk 2 above)
Comment (Denijar) [reply client/src/components/SearchBar/style.module.css:1]: Sure happy to have a look. Can we get some documentation on this styling method tho? This is not something taught in 750

File: client/src/components/SearchBar/index.js (hunk 1 above)
Comment (sannan06) [reply client/src/components/SearchBar/index.js:1]: Not personally a fan of this approach; makes it nearly impossible to search for component files by their name or infer a file's contents from its name but I'll concede it if it's what the team wants

File: client/src/components/SearchBar/style.module.css (hunk 2 above)
Comment (sannan06) [reply client/src/components/SearchBar/style.module.css:1]: You simply change the `.css` extension to `.scss`. All CSS is valid SCSS so the content does not have to change

File: client/src/components/SearchBar/index.js (hunk 1 above)
Comment (Denijar) [reply client/src/components/SearchBar/index.js:1]: Oh okay shoot, I mentioned this file structure a while ago and only got positive reactions so thought it was fine. Feel free to make a refactoring issue. I think for the small size of our repo, perhaps cleanliness of the import statements outweighs searchability? I think for this PR I will sick with the index.js naming for the sake of getting the code in.

File: client/src/components/SearchBar/style.module.css (hunk 2 above)
Comment (Denijar) [reply client/src/components/SearchBar/style.module.css:1]: Easy then. This needs to be documented though in the wiki.

**Review by sannan06 — APPROVED** [review]

**Review by Jigao-Zeng — APPROVED** [review]
Good work on the search bar.

**Pull request merged**